fhir-notification-kafka/src/main/java/com/ibm/fhir/notifications/kafka/impl/FHIRNotificationKafkaPublisher.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2016,2019
+ * (C) Copyright IBM Corp. 2016, 2020
```

Comment:
Any specific reasons other than whitespace for this update?

Reply:
nope.  i had the edits sitting around and thought i might as well include them.  happy to remove if you prefer.

Reply:
I didn't want to miss anything :) or the commit to.  This is A-OK!